packages/gp2-components/src/organisms/ProjectDetailHeader.tsx

```diff
@@ -75,69 +77,82 @@ const ProjectDetailHeader: React.FC<ProjectDetailHeaderProps> = ({
   opportunitiesLink,
   isAdministrator,
   outputsTotal = 0,
-}) => (
-  <header css={detailHeaderStyles}>
-    {opportunitiesLink && (
-      <Card accent="information" padding={false}>
-        <div css={opportunitiesCardStyles}>
-          <div>{informationIcon}</div>
-          <div css={cardTextContainerStyles}>
-            <Subtitle noMargin>Opportunities Available</Subtitle>
-            <Paragraph noMargin>
-              This project is currently looking for additional team members.
-            </Paragraph>
-            <Link href={opportunitiesLink}>
-              <span css={css({ display: 'inline-flex' })}>
-                Read more <span css={infoIconStyles}>{externalLinkIcon}</span>
-              </span>
-            </Link>
+  upcomingTotal = 0,
+  pastTotal = 0,
+}) => {
+  const route = gp2Routing.projects({}).project({ projectId: id });
```

Comment:
we are only using it in one of the tablink, either we use it more or remove it 😄

Reply:
We use it in 5 places.